src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -77,6 +77,13 @@ def create_webapp(resource_group_name, name, plan, runtime=None, startup_file=No
             raise CLIError("Runtime '{}' is not supported. Please invoke 'list-runtimes' to cross check".format(runtime))  # pylint: disable=line-too-long
         match['setter'](match, site_config)
 
+    if not is_linux:
+        site_config.app_settings.append(NameValuePair("WEBSITE_NODE_DEFAULT_VERSION", "6.9.1"))
```

Comment:
Why we need this? Please note, the `_StackRuntimeHelper._load_stacks` has the logic to turn it on only when you use `node` through the `--runtime` flag

Reply:
If runtime isn't specified we still want this app setting defined. It's the only app stack version that isn't defined in our site_config yet.

Reply:
Got it, then shall we apply the the condition of `not runtime and not deployment_container_image_name`?